nostr/relay.py

```diff
@@ -37,8 +37,8 @@ def __init__(
             on_error=self._on_error,
             on_close=self._on_close)
 
-    def connect(self, ssl_options: dict=None):
-        self.ws.run_forever(sslopt=ssl_options)
+    def connect(self, ssl_options: dict=None, proxy: dict=None):
+        self.ws.run_forever(sslopt=ssl_options,http_proxy_host=proxy.get('host'), http_proxy_port=proxy.get('port'))
```

Comment:
This will raise an `AttributeError` if `proxy` is `None` because you can't do `.get()` on a `NoneType`.

```suggestion
        self.ws.run_forever(
            sslopt=ssl_options,
            http_proxy_host=None if proxy is None else proxy.get('host'), 
            http_proxy_port=None if proxy is None else proxy.get('port')
        )
```